pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -2570,6 +2570,8 @@ func buildListenerEnvWithVirtualServices(services []*model.Service, virtualServi
 		IstioConfigStore: configStore,
 		Watcher:          mesh.NewFixedWatcher(&m),
 	}
+	env.JwtKeyResolver = model.NewJwksResolver(model.JwtPubKeyEvictionDuration, model.JwtPubKeyRefreshInterval, model.JwtPubKeyRetryInterval)
+	defer env.JwtKeyResolver.Close()
```

Comment:
this is closed just after the following return?

Reply:
Thanks! Fixed all brutal defers in tests.